src/api/app/models/owner_search/assignee.rb

```diff
@@ -94,6 +94,7 @@ def parse_binary_info(b, prj)
 
       package_name = b['package']
       package_name.gsub!(/\.[^.]*$/, '') if prj.is_maintenance_release?
+      package_name = Package.striping_multibuild_suffix(package_name)
```

Comment:
It could for sure be extracted to a method, where you give the` b['package']` and `pkg` and returns the results of `prj.packages.find_by_name`

Reply:
All of this should be refactored deeply, yes.